libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -290,6 +306,44 @@ TEST_F(CppInterface, DisassemblyOption) {
   EXPECT_THAT(result_from_cloned_options.GetData(), HasSubstr("MemoryModel"));
 }
 
+TEST_F(CppInterface, GenerateDebugInfoBinary) {
+  options_.SetGenerateDebugInfo();
+  // The output binary should contain the name of the vector: debug_info_sample as char
+  // array.
+  EXPECT_THAT(CompilationOutput(kMinimalDebugInfoShader,
+                                shaderc_glsl_vertex_shader, options_),
+              HasSubstr("debug_info_sample"));
```

Comment:
Right.  I'll back down on this for now.
Maybe add a TODO with the idea for how to strengthen the test.